functions.gallery.php

```diff
@@ -35,7 +44,12 @@ function wp_enqueue_media() {
 
 	/**
 	 * Outputs a view template which can be used with wp.media.template
-	 */
+	 *
+	 * @Since 2.5.1
+	 *
+	 * @param array  $instance An array of gallery types.
```

Comment:
This appears to be a string, not an array. It only contains the default gallery type, `default`.